curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/step-033.md

```diff
@@ -1,28 +1,32 @@
 ---
-id: 60a3e3396c7b40068ad6998a
+id: 60a3e3396c7b40068ad6998b
 title: Step 33
 challengeType: 0
 dashedName: step-33
 ---
 
 # --description--
 
-Center the `.three` element on the canvas by setting its `margin` to `auto`.
+It's helpful to have your margins push in one direction.
+
+In this case, the bottom margin of the `.one` element pushes `.two` down 20 pixels.
+
+In `.two`, add `margin: 0 auto 20px;` to set its top margin to 0, center it horizontally, and set its bottom margin to 20 pixels.
```

Comment:
```suggestion
In the `.two` selector, use `margin` shorthand property to set top margin to `0`, horizontal margin to `auto`, and bottom margin to `20px`. This will remove its top margin, horizontally center it, and set its bottom margin to 20 pixels.
```